dimod/core/sampler.py

```diff
@@ -23,6 +23,10 @@
 or :meth:`~.Sampler.sample_qubo`. The :class:`.Sampler` class provides the complementary
 methods as mixins and ensures consistent responses.
 
+All sample methods must be able to accept unknown kwargs with a raised warning.
```

Comment:
What about a slight rephrasing to something like, 
"Implemented sample methods must accept, and warn on, unknown keyword arguments ``**kwargs``.

Reply:
That sounds better, thanks!